text_formatter_test.go

```diff
@@ -72,36 +60,13 @@ func TestEscaping_DefaultQuoteCharacter(t *testing.T) {
 	}
 }
 
-func TestEscaping_CustomQuoteCharacter(t *testing.T) {
-	tf := &TextFormatter{DisableColors: true}
-
-	testCases := []struct {
-		value     string
-		expected  string
-		quoteChar string
-	}{
-		{`ba"r`, `ba"r`, `'`},
-		{`ba'r`, `ba\'r`, `'`},
-		{`ba^r`, `ba\^r`, `^`},
-		{`ba'r`, `ba'r`, `^`},
-	}
-
-	for _, tc := range testCases {
-		tf.QuoteCharacter = tc.quoteChar
-		b, _ := tf.Format(WithField("test", tc.value))
-		if !bytes.Contains(b, []byte(tc.expected)) {
-			t.Errorf("escaping expected for %q (result was %q instead of %q)", tc.value, string(b), tc.expected)
-		}
-	}
-}
-
 func TestTimestampFormat(t *testing.T) {
 	checkTimeStr := func(format string) {
 		customFormatter := &TextFormatter{DisableColors: true, TimestampFormat: format}
 		customStr, _ := customFormatter.Format(WithField("test", "test"))
 		timeStart := bytes.Index(customStr, ([]byte)("time="))
 		timeEnd := bytes.Index(customStr, ([]byte)("level="))
-		timeStr := customStr[timeStart+5+len(customFormatter.QuoteCharacter) : timeEnd-1-len(customFormatter.QuoteCharacter)]
+		timeStr := customStr[timeStart+5+len("\"") : timeEnd-1-len("\"")]
```

Comment:
No particular reason, no. This is in tests though, there is no huge speed issues.